src/BrightScriptCommenter.ts

```diff
@@ -318,41 +316,22 @@ export class BrightScriptCommenter {
    * Gets the type name for the given type
    * Defaults to "Dynamic" if it can't decide
    *
-   * @param { number | bs.Token } type id or Type Token
-   * @param { bs.Token } token actual token for the type name
+   * @param {bs.BscType } type to get name for
    * @returns {string} the name for the type id given
    */
-  private getTypeName(type: number | bs.Token, token?: bs.Token): string {
-    let typeName = this.useDynamicIfNoTypeGiven ? "Dynamic" : "";
+  private getTypeName(type: any): string {
```

Comment:
getTypeName is much simplified... the Brighterscript project does not export "BscType", so that's why it's set as "any"